src/Resolvers/__tests__/fixtures/data.ts

```diff
@@ -20,6 +20,7 @@ export const mockCollectionRepository = plainToClass(Collection, [
     show_on_editorial: false,
     price_guidance: null,
     is_featured_artist_content: true,
+    linkedCollections: [{ id: "abc", members: [{ id: 3 }] }],
```

Comment:
#minor Not a big deal, but here and elsewhere I was a little confused by the `id` attribute, since that's not part of the schema or current data (although `name` is). Maybe worth a cleanup to save us future confusion?

Reply:
Updated!